src-built-in/components/authentication/defaultAuthentication.html

```diff
@@ -118,35 +117,96 @@
 
 <script>
 	var inLogin = true;
-	//Finsemble will never come online for the authentication component and we don't want to alarm anyone.
-	FSBL.displayStartupTimeoutError = false;
+	/************************************************
+	 * 			SAMPLE AUTHENTICATION
+	 ************************************************/
+	//On ready, check to see if the user has a valid session
+	FSBL.Clients.RouterClient.onReady(() => {
+		checkAuthorizationStatus();
+	});
+
+	$('#authAction').click(function (e) {
+		var text = inLogin ? "Sign Up" : "Login"
+		var actionLink = inLogin ? "Login" : "Sign Up";
+		inLogin = !inLogin;
+		$('#submitButton').html(text);
+		$('#authAction').html(actionLink);
+	});
+
+	document.body.addEventListener('keydown', handleKeydown);
+
+	//Submits credentials on enter, closes on quit.
+	function handleKeydown(e) {
+		if (e.code === 'Enter' && e.shiftKey === false) {
+			processAuthInput();
+		}
+
+		if (e.code === 'Escape') {
+			quit();
+		}
+	}
+
+	//Here, you may want to hit a server and request the user's session information. If the session is valid, log them in automatically. This sample code assumes that they are not logged in and just shows the authentication page.
+	function checkAuthorizationStatus() {
+		FSBL.Clients.WindowClient.finsembleWindow.show();
+	}
+
+	//Dummy function that just dumbly accepts whatever is in the form.
+	function processAuthInput() {
+		var username = document.getElementById("username").value;
+		var password = document.getElementById("password").value;
+		// real authentication might use BasicAuth, Digest Auth, or pass off to authentication server which redirects back when authenticated
+		// below is a dummy example that just accepts credentials from the form and publishes them out.
+		var data = { username: username, password: password }
+		FSBL.Clients.WindowClient.finsembleWindow.hide();
+		//In the real world, you'd get this from a server. Send joe's credentials to a server, and get back entitlements/basic config. For this example, we just accept the credentials.
+		publishCredentials(data)
+	}
+
+	//Pass credentials to the application.
+	function publishCredentials(user) {
+		FSBL.Clients.AuthenticationClient.publishAuthorization(user.username, user);
+		FSBL.Clients.WindowClient.close({ removeFromWorkspace: false, closeWindow: true });
+	}
+
+	//CLose app when the X is clicked.
 	function quit() {
 		FSBL.shutdownApplication();
 	}
-	//Check to see if we already have a good session
-	function checkSession() {
-		fetch("/isAuthenticated", {
-			method: "GET",
-			credentials: 'include'
-		})
-			.catch((reason) => {
-				console.warn("Fail Auth Get", reason);
+
+
+
+
+
+
+
+
+
+
+
+	/************************************************
```

Comment:
@carterbs We should either document or remove the dynamic config code. I would vote to document it.